titus-server-master/src/main/java/com/netflix/titus/master/jobactivity/service/JobActivityPublisherConfiguration.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.master.jobactivity.service;
+
+import com.netflix.archaius.api.annotations.Configuration;
+import com.netflix.archaius.api.annotations.DefaultValue;
+
+@Configuration(prefix = "titusMaster.jobActivityPublisher")
+public interface JobActivityPublisherConfiguration {
+
+    @DefaultValue("true")
+    boolean getJobActivityPublisherEnabled();
```

Comment:
This property should be in `FeatureActivationConfiguration`

Reply:
Done, moved to `FeatureRolloutPlan`. That'll allow more incremental on-boarding and ramp up of load.